pkgs/applications/audio/mpg123/default.nix

```diff
@@ -43,8 +65,8 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Fast console MPEG Audio Player and decoder library";
     homepage = "https://mpg123.org";
-    license = licenses.lgpl21;
-    maintainers = [ maintainers.ftrvxmtrx ];
+    license = licenses.lgpl21Only;
```

Comment:
Skimmed the docs & source and it only mentions "LGPL v2.1" afaict.

Reply:
Which is lgpl21Only.

Reply:
Hence the change, yes.